src/wallet/wallet.cpp

```diff
@@ -7831,6 +7836,23 @@ bool ZTXOSelector::SelectsTransparent() const {
         [](const AccountZTXOPattern& acct) { return acct.IncludesP2PKH() || acct.IncludesP2SH(); }
     }, this->pattern);
 }
+bool ZTXOSelector::SelectsTransparentCoinbase() const {
+    return std::visit(match {
+        [](const CKeyID& keyId) { return true; },
+        [](const CScriptID& scriptId) { return true; },
+        [](const libzcash::SproutPaymentAddress& addr) { return false; },
+        [](const libzcash::SproutViewingKey& vk) { return false; },
+        [](const libzcash::SaplingPaymentAddress& addr) { return false; },
+        [](const libzcash::SaplingExtendedFullViewingKey& vk) { return false; },
+        [](const libzcash::UnifiedAddress& ua) {
+            return ua.GetP2PKHReceiver().has_value() || ua.GetP2SHReceiver().has_value();
+        },
+        [](const libzcash::UnifiedFullViewingKey& ufvk) { return ufvk.GetTransparentKey().has_value(); },
+        [](const AccountZTXOPattern& acct) {
+            return (acct.IncludesP2PKH() || acct.IncludesP2SH()) && acct.GetAccountId() != ZCASH_LEGACY_ACCOUNT;
+        }
+    }, this->pattern);
+}
```

Comment:
@nuttycom I think this was added to this PR by accident. I’ve removed it from later changes, but wanted to mention it here so you can recover it. (Although, I’m guessing it’s just been extracted from the WalletTxBuilder work)

Reply:
Nevermind. Now that CI is failing I see why this is needed 😅